packages/react-scripts/config/pnpTs.js

```diff
@@ -1,43 +0,0 @@
-// @remove-on-eject-begin
-/**
- * Copyright (c) 2015-present, Facebook, Inc.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-// @remove-on-eject-end
-'use strict';
-
-const { resolveModuleName } = require('ts-pnp');
```

Comment:
Is this no longer needed?

Reply:
pnp support should be builtin webpack 5 and the fork-ts-checker - @iansu do you have a pnp test project to QA if it's still working as expected? (the pnp test have been failing the past year, so not sure if pnp is actually working still)

Reply:
@iansu closing this for now but let's QA pnp in the alpha (maybe have a look into the pnp tests if it can be fixed?)